### PR TITLE
Add initial version of the build_and_test script.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by jverkoey** [comment]
Gentle ping. This is blocking rollout of kokoro to other repos.

File: build_and_test.sh [hunk 1]
```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Copyright 2017-present The kokoro iOS runner Authors. All Rights Reserved.
```
Comment (ajsecord) [inline build_and_test.sh]: Capitalization on the authors seems wrong.

File: build_and_test.sh [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  : # Local run - nothing to do.
+else
+  gem install xcpretty --no-rdoc --no-ri --no-document --quiet
+
+  # Move into our cloned repo
+  cd github/repo
+fi
+
+# Runs a clean build + tests using xcodebuild on the active Xcode installation.
+run() {
```
Comment (ajsecord) [inline build_and_test.sh]: Usually you put your functions up as high as possible in the file.

**Review by ajsecord — APPROVED** [review]

File: build_and_test.sh (hunk 1 above)
Comment (jverkoey) [reply build_and_test.sh]: Done.

File: build_and_test.sh (hunk 2 above)
Comment (jverkoey) [reply build_and_test.sh]: Done.

**Pull request merged**